ecs-agent/tmds/handlers/fault/v1/handlers/handlers_test.go

```diff
@@ -127,6 +127,102 @@ func TestFaultPacketLossFaultPath(t *testing.T) {
 	assert.Equal(t, "/api/{endpointContainerIDMuxName:[^/]*}/fault/v1/network-packet-loss", NetworkFaultPath(types.PacketLossFaultType))
 }
 
+// testNetworkFaultInjectionCommon will be used by unit tests for all 9 FIS Network Fault APIs.
```

Comment:
Let's not use FIS in the code or comments and use fault injection instead. Because we are trying to make these new APIs be generic.

Reply:
Addressing in the next revision